docs/rules/prefer-arrow-callback.md

```diff
@@ -1,15 +1,22 @@
-# Suggest using arrow functions as callbacks. (prefer-arrow-callback)
+# Suggest using ES6 arrow functions to describe callbacks (prefer-arrow-callback)
 
-Arrow functions are suited to callbacks, because:
+Arrow functions can be an attractive alternative to function expressions, when describing callbacks or function arguments.
 
-- `this` keywords in arrow functions bind to the upper scope's.
-- The notation of the arrow function is shorter than function expression's.
+For example, arrow functions are bound to their surrounding scope/context automatically. This is an upgrade from using `bind()` and `this` to explicitly bind a function expression, which was the pre-ES6 standard for achieving similar scoping behavior.
+
+Additionally, arrow functions are:
+
+- less verbose and easier to reason about.
+
+- more robust and lexically bound regardless of where or when they are eventually invoked.
 
 ## Rule Details
 
-This rule is aimed to flag usage of function expressions in an argument list.
+This rule identifies function expressions being used as callbacks or function arguments, where upgrading to an arrow function would achieve identical results.
```

Comment:
"where upgrading to an arrow function" doesn't feel like right here. How about "where using an arrow function" instead?